src/diffusers/dependency_versions_table.py

```diff
@@ -40,7 +40,7 @@
     "tensorboard": "tensorboard",
     "torch": "torch>=1.4",
     "torchvision": "torchvision",
-    "transformers": "transformers>=4.25.1",
+    "transformers": "transformers>=4.41.2",
```

Comment:
Is this still needed?

Reply:
This is reformatted automatically after `make style`

Reply:
No I mean this is not just a formatting change, it’s changing the version. We have had this change merged recently from another PR actually.

Reply:
@haofanwang I think you need to merge main (these change are already in main)

https://github.com/huggingface/diffusers/blob/main/src/diffusers/dependency_versions_table.py